### ISSUE TITLE
Add subscript and superscript support

### ISSUE THREAD

**Issue by dlemmermann** [gluonhq/rich-text-area#79]
The editor should allow to use subscript and superscript so that (for example) footnote numbers can be added or simple mathematical equations.

**Comment by sten-navie** [comment]
I agree that this would be nice, but if we have to prioritise this is a feature that we can leave for later.

**Comment by dlemmermann** [comment]
+1 ... just put it there so we do not forget
